templates/web/base/report/update/form_state_checkbox.html

```diff
@@ -1,5 +1,7 @@
 [% IF (problem.is_fixed OR problem.is_closed) AND ((c.user_exists AND c.user.id == problem.user_id) OR alert_to_reporter) %]
 
+    [% RETURN IF c.cobrand.reopening_disallowed(problem) ~%]
```

Comment:
Should this change be included in the `fixmystreet.com` version of this template or is it a deliberate thing to limit that flag to cobrands?